ProjFS.Mac/Scripts/Build.sh

```diff
@@ -12,7 +12,19 @@ PACKAGES=$ROOTDIR/packages
 
 PROJFS=$SRCDIR/ProjFS.Mac
 
-xcodebuild -configuration $CONFIGURATION -project $PROJFS/PrjFS.xcodeproj build -scheme 'Build All' -derivedDataPath $ROOTDIR/BuildOutput/ProjFS.Mac/Native || exit 1
+xcodebuild -configuration $CONFIGURATION -project $PROJFS/PrjFS.xcodeproj  -scheme 'Build All' -derivedDataPath $ROOTDIR/BuildOutput/ProjFS.Mac/Native build || exit 1
+
+if !(gem list --local | grep xcpretty); then
+   sudo gem install xcpretty
```

Comment:
This may prompt for a password if required.  It would only do this once to install the gem, once installed no more prompts.

Reply:
If I ran this as a user and it asked me for a password, I'd start digging through the shell source to find out what it was trying to do. Perhaps we can echo something like "Attempting to run 'sudo gem install xcpretty'. This may ask you for your password to gain admin privileges."